server/db.js

```diff
@@ -0,0 +1,27 @@
+const sqlite3 = require('sqlite3').verbose();
+const db = new sqlite3.Database('./server/db/app.db');
+
+// Create a table just for once
+const initialise = db.run(`CREATE TABLE IF NOT EXISTS user (
```

Comment:
This operation doesn't need to be defined as initialize.

If anything, this db.run function should be wrapped in an async function that is executed in this file

Reply:
Got it !
The variable is no longer used and `db.run` is run properly so async wont be applied to this code for this time.
Apart from that your point taken.